.ci/scripts/resolve-dra-manifest.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+ARTIFACT="${ARTIFACT:-$1}"
+BRANCH="${BRANCH:-$2}"
+
+curl -sS https://artifacts-snapshot.elastic.co/$ARTIFACT/latest/$BRANCH.json \
+  | jq -r '.build_id' \
+  | xargs
```

Comment:
What's the point of `xargs` here?

Reply:
it was required in earlier versions of that script. removed